core/src/main/java/io/questdb/cairo/sql/PageAddressCacheRecord.java

```diff
@@ -82,7 +82,7 @@ public long getBinLen(int columnIndex) {
             return NullMemoryMR.INSTANCE.getBinLen(0);
         }
         final long indexPageAddress = pageAddressCache.getIndexPageAddress(frameIndex, columnIndex);
-        final long offset = Unsafe.getUnsafe().getLong(indexPageAddress + rowIndex * Long.BYTES);
+        final long offset = Unsafe.getUnsafe().getLong(indexPageAddress + (rowIndex << 3));
```

Comment:
nit: these are equivalent

Reply:
Not from bytecode perspective. JIT compiler will have to kick in and replace multiplication with bitshift.

A side note: with division it's not even possible. `l >>> 1` and `l / 2` aren't equivalent due to behavior on negative numbers, so we should prefer the former on hot path code.